### PR TITLE
NETOBSERV-933: Update uninstallation steps

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.10+
Issue:

https://issues.redhat.com/browse/NETOBSERV-933

Link to docs preview:

https://57468--docspreview.netlify.app/openshift-enterprise/latest/networking/network_observability/installing-operators.html#network-observability-operator-uninstall_network_observability
QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by jotak — APPROVED** [review]

**Review by memodi — APPROVED** [review]
/qe-approved

File: modules/network-observability-operator-uninstall.adoc [hunk 1]
```diff
@@ -9,9 +9,17 @@ You can uninstall the Network Observability Operator using the {product-title} w
...
 . Uninstall the Network Observability Operator.
 .. Navigate back to the *Operators* -> *Installed Operators* area. 
-.. Click the options menu {kebab} next to the  *Network Observability Operator* and select *Uninstall Operator*.
\ No newline at end of file
+.. Click the options menu {kebab} next to the  *Network Observability Operator* and select *Uninstall Operator*.
+. Remove the `FlowCollector` custom resource definition (CRD).
+.. Navigate to *Administration* -> *CustomResourceDefinitions*. 
+.. Look for *FlowCollector* and click the options menu {kebab}. 
+.. Select *Delete CustomResourceDefinition*.
+[IMPORTANT]
```
Comment (JoeAldinger) [inline modules/network-observability-operator-uninstall.adoc:22]: It doesn't look like the IMPORTANT section is rendering correctly. I think you need a space between line 21 and 22.

**Review by JoeAldinger — COMMENTED** [review]
Small change to get the IMPORTANT to render correctly. Otherwise, lgtm

File: modules/network-observability-operator-uninstall.adoc [hunk 2]
```diff
@@ -9,9 +9,18 @@ You can uninstall the Network Observability Operator using the {product-title} w
...
\ No newline at end of file
+.. Click the options menu {kebab} next to the  *Network Observability Operator* and select *Uninstall Operator*.
+. Remove the `FlowCollector` custom resource definition (CRD).
+.. Navigate to *Administration* -> *CustomResourceDefinitions*. 
+.. Look for *FlowCollector* and click the options menu {kebab}. 
+.. Select *Delete CustomResourceDefinition*.
++
+[IMPORTANT]
+====
+The Loki Operator and Kafka remain if they were installed and must be removed separately. Additionally, you might have remaining data stored in object store, and persistent volume that must be removed.
```
Comment (stevsmit) [inline modules/network-observability-operator-uninstall.adoc:25]: Requesting a revision on the note. Maybe something like... 

. Additionally, you might have the remaining data stored in an object store, and a persistent volume that must be removed.

Comment (skrthomas) [reply modules/network-observability-operator-uninstall.adoc:25]: Agree, this is 100% missing articles THANK YOU! Adding a/an

**Pull request merged**